Framework/Crystal/src/SaveLauenorm.cpp

```diff
@@ -161,7 +161,7 @@ void SaveLauenorm::exec() {
   double minD = EMPTY_DBL();
   for (int wi = 0; wi < ws->getNumberPeaks(); wi++) {
 
-    Peak &p = peaks[wi];
+    Peak const &p = peaks[wi];
```

Comment:
To make it consistent with const style used elsewhere in this pr
```suggestion
    const Peak &p = peaks[wi];
```